JavaIntegration/src/main/java/org/openzen/zencode/java/ScriptingEngine.java

```diff
@@ -63,7 +63,7 @@ public ScriptingEngine(ScriptingEngineLogger logger) {
 	}
     
     public ScriptingEngine(ScriptingEngineLogger logger, Function<String, InputStream> resourceGetter) {
-        this(logger, resourceGetter, "stdlib"/*, "math", "collections", "uuid"*/);
+        this(logger, resourceGetter, "stdlib", "math", "collections", "uuid");
```

Comment:
Works in the ZC context, but in CrT I get an error when I use more than one stdlib module, gonna have to investigate further at some point...